fam_create_region.html

```diff
@@ -142,20 +147,39 @@ <h2>Input Arguments</h2>
 	</table>
 	<p>&nbsp;</p>
 	<h2>Return Values</h2>
-	<p>Descriptor to the created region. Throws an exception on error.</p>
-	<h2>Exceptions</h2>
+	<p>Descriptor to the created region. Throws <code>Fam_Exception</code> on error.</p>
+	<h2>Fam error numbers</h2>
 	<table>
 	<tbody>
-	<tr><th>Exception</th><th>Description</th></tr>
+	<tr><th>Error</th><th>Description</th></tr>
 	<tr><td class="code">FAM_ERR_NOPERM</td><td>Caller does not have access rights</td></tr>
-	<tr><td class="code">FAM_ERR_ALREADYEXIST</td><td>the region name is already present in FAM</td></tr>
-	<tr><td class="code">FAM_ERR_GRPC</td><td>there is a communication error with memoryserver</td></tr>
-	<tr><td class="code">FAM_ERR_RPC_CLIENT_NOTFOUND</td><td>Memory server initialization failure</td></tr>
+	<tr><td class="code">FAM_ERR_ALREADYEXIST</td><td>Region name is already present in FAM</td></tr>
+	<tr><td class="code">FAM_ERR_NAME_TOO_LONG</td><td>Region name too long.</td></tr>
+	<tr><td class="code">FAM_ERR_NOT_CREATED</td><td>Region creation in FAM failed.</td></tr>
+	<tr><td class="code">FAM_ERR_RPC</td><td>Communication error from grpc layer.</td></tr>
+	<tr><td class="code">FAM_ERR_RPC_CLIENT_NOTFOUND</td><td>RPC service not available.</td></tr>
+	<tr><td class="code">FAM_ERR_METADATA</td><td>Metadata service error.</td></tr>
+	<tr><td class="code">FAM_ERR_MEMORY</td><td>Memory service error.</td></tr>
+	<tr><td class="code">FAM_ERR_RESOURCE</td><td>Resource not available.</td></tr>
 	</tbody></table>
 	<h2>Notes</h2>
 	<p>Note that the system may round up the actual size of the region
 		to align memory boundaries. Currently the maximum size of a region is
-		limited to 16 EiB by the API.</p>
+		limited to 16 EiB by the API. See <a href="limitations.html">Design Choices</a> for the current design limits within the OpenFAM reference implementation.
+		</p>
+	<p>Fam_Region_Descriptor provides a method to retrieve an opaque read-only data structure called 
+	<code>Fam_Global_Descriptor</code> which represents the region in FAM. <code>Fam_Global_Descriptor</code>
+	can be freely copied, and shared across processing nodes by the program, and is portable
+	across operating system and program instances.</p>
+<pre>
+// Fam_Global_Descriptor Fam_Region_Descriptor::get_global_descriptor();
+Fam_Global_Descriptor gDescriptor = regionDescriptor->get_global_descriptor();
+</pre>
+	<p>Fam_Region_Descriptor can be instantiated from Fam_Global_Descriptor using the following constructor.</p>
```

Comment:
Line 178, insert code /code blocks around Fam_Region_Descriptor/Fam_Global_Descriptor for consistent formatting.